### DISCUSSION TITLE
How do you change the ID of a post?

### DISCUSSION THREAD

**Discussion by Oo1Toyo1oO** [GetPublii/Publii#1545] [Help]
Hey! I deleted the first post i made as it was just a shitpost to test how publii works, but now the ID 1 doesn't exist anymore and the 1st post has the ID 2, as a perfectionist this really annoys me, is there a way to move all of the ids one number down?

![image](https://github.com/GetPublii/Publii/assets/87552280/414d0636-7664-4ef2-bd19-39b154959fb8)

**Comment by bobmitro** [comment]
This can be done by editing the database.

**Comment by dziudek** [comment]
If it is a new website - please start it from scratch. Changing via database ID can lead to issues with references to the post. Also database has auto-increment mechanism which should be also updated after such change
